docs/src/api/class-page.md

```diff
@@ -2545,13 +2545,22 @@ Video object associated with this page.
   - `height` <[int]> page height in pixels.
 
 ## method: Page.waitForClose
```

Comment:
@yury-s I think you ignore the `async` method in Java generation right, so @kblok, I think you can just mark this as `async`.